Sources/Subs.php

```diff
@@ -6613,6 +6613,8 @@ function build_query_board($userid)
 	else
 		$query_part['query_wanna_see_board'] = '(' . $query_part['query_see_board'] . ' AND b.id_board NOT IN (' . implode(',', $ignoreboards) . '))';
 
+	$query_part['query_see_message'] = str_replace('b.', 'm.', $query_part['query_see_board']);
+
```

Comment:
For completeness, it would be nice to have `query_see_topic_board`, too. Then we could leave the instances of `{db_prefix}topics AS t` as they are. 

I know this wouldn't make any functional difference, but it would make it easier for future developers to understand what is going on.